pandas/tests/extension/test_categorical.py

```diff
@@ -229,21 +228,26 @@ def test_consistent_casting(self, dtype, expected):
 
 
 class TestArithmeticOps(base.BaseArithmeticOpsTests):
-    def test_arith_frame_with_scalar(self, data, all_arithmetic_operators):
+    def test_arith_frame_with_scalar(self, data, all_arithmetic_operators, request):
         # frame & scalar
         op_name = all_arithmetic_operators
-        if op_name != "__rmod__":
-            super().test_arith_frame_with_scalar(data, all_arithmetic_operators)
-        else:
-            pytest.skip("rmod never called when string is first argument")
-
-    def test_arith_series_with_scalar(self, data, all_arithmetic_operators):
-
+        if op_name == "__rmod__":
+            request.node.add_marker(
+                pytest.mark.xfail(
```

Comment:
Is this change related? (I wouldn't know if an xfail is more appropriate here than a skip)

Reply:
Related to #38904 that this PR is trying to close, but not to the other skip/xfail in this PR. Will split off into a separate PR if desired.

This test is doing op(Series, const) expecting it to raise, where Series consists of and const is a string.  There is no rmod in the python library `operator`, and so the test is instead doing `lambda x, y: mod(const, Series)` . This doesn't raise.

This is always expected to "fail" (meaning not raise), and I think we'd want to know if it didn't. This is why I went with xfail over skip here.